networkx/algorithms/approximation/clique.py

```diff
@@ -40,6 +40,14 @@ def maximum_independent_set(G):
     iset : Set
         The apx-maximum independent set
 
+    Examples
+    --------
+    >>> import networkx as nx
```

Comment:
```suggestion
```

The `import networkx as nx` is automatically included for all docstring examples and should not be explicitly added.